internal/service/fw/model_atcfw_internal_domains.go

```diff
@@ -0,0 +1,144 @@
+	"created_time":     timetypes.RFC3339Type{},
+	"description":      types.StringType,
+	"id":               types.Int64Type,
+	"internal_domains": types.ListType{ElemType: types.StringType},
+	"is_default":       types.BoolType,
+	"name":             types.StringType,
+	"tags":             types.MapType{ElemType: types.StringType},
+	"updated_time":     timetypes.RFC3339Type{},
+}
+
+var AtcfwInternalDomainsResourceSchemaAttributes = map[string]schema.Attribute{
+	"created_time": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "The time when this Internal Domain lists object was created.",
+	},
+	"description": schema.StringAttribute{
+		Optional:            true,
+		Default:             stringdefault.StaticString(""),
+		Computed:            true,
+		MarkdownDescription: "The brief description for the internal domain lists .",
+	},
+	"id": schema.Int64Attribute{
+		Computed: true,
+		PlanModifiers: []planmodifier.Int64{
+			int64planmodifier.UseStateForUnknown(),
+		},
+		MarkdownDescription: "The Internal Domain object identifier.",
+	},
+	"internal_domains": schema.ListAttribute{
+		ElementType:         types.StringType,
+		Optional:            true,
+		MarkdownDescription: "The list of internal domains, should be unique to each other and has to be read-only from the API level.",
+	},
+	"is_default": schema.BoolAttribute{
+		Computed:            true,
+		MarkdownDescription: "True if name is 'Default Bypass Domains/CIDRs' otherwise false.",
+	},
+	"name": schema.StringAttribute{
+		Optional: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
+		},
+		MarkdownDescription: "The name of the internal domain lists.",
+	},
+	"tags": schema.MapAttribute{
+		ElementType:         types.StringType,
+		Optional:            true,
+		MarkdownDescription: "Enables tag support for resource where tags attribute contains user-defined key value pairs",
+	},
+	"updated_time": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "The time when this Internal domain lists object was last updated.",
+	},
+}
+
+func ExpandAtcfwInternalDomains(ctx context.Context, o types.Object, diags *diag.Diagnostics) *fw.AtcfwInternalDomains {
+	if o.IsNull() || o.IsUnknown() {
+		return nil
+	}
+	var m AtcfwInternalDomainsModel
+	diags.Append(o.As(ctx, &m, basetypes.ObjectAsOptions{})...)
+	if diags.HasError() {
+		return nil
+	}
+	return m.Expand(ctx, diags)
+}
+
+func (m *AtcfwInternalDomainsModel) Expand(ctx context.Context, diags *diag.Diagnostics) *fw.AtcfwInternalDomains {
+	if m == nil {
+		return nil
+	}
+	to := &fw.AtcfwInternalDomains{
+		Description:     flex.ExpandStringPointer(m.Description),
+		InternalDomains: flex.ExpandFrameworkListString(ctx, m.InternalDomains, diags),
+		IsDefault:       flex.ExpandBoolPointer(m.IsDefault),
```

Comment:
You shouldn't expand read only properties.

Reply:
fixed